src/manifest/getManifest.js

```diff
@@ -32,7 +31,7 @@ export const compareManifestTypes = (a, b) => {
 }
 
 /**
- * @param {{ task: import('../types.js').ScheduledTask, tasks: TaskGraph, manifestConstructor: ManifestConstructor }} param0
+ * @param {{ task: import('../types.js').ScheduledTask, tasks: import('../TaskGraph.js').TaskGraph, manifestConstructor: import('./computeManifest.js').ManifestConstructor }} param0
```

Comment:
I had to refactor this bit (and other similar occurrences) because we were getting lint errors about unused variables. We can avoid doing that and use JSDoc type definitions as it was before this change, but we'll need to install jsdoc eslint plugin. Any strong opinion on this style of writing type defs?

Reply:
nah this is fine 👍🏼